README.md

```diff
@@ -422,7 +422,7 @@ SCIMetacelloLoadSpec {
   #passwordEnvVar : 'GITHUB_TOKEN',                   // Environment variable containing the password used for authentication
   #useLatestMetacello : true,                         // Upgrade Metacello before loading
   #version : '1.0.0'                                  // Define MC version (for MC Configurations only)
-  #registerInIceberg : true				      // Pharo Only | Register the tested repository with Iceberg
+  #registerInIceberg : true			// Pharo Only | Register the tested repository with Iceberg
```

Comment:
Sorry for the hassle. My last change request didn't quite show what I wanted it to, which is to align the comments properly. I think the issue is that there are tabs in that line, the other lines contain only spaces. Let me try again:
```suggestion
  #registerInIceberg : true                           // Pharo Only | Register the tested repository with Iceberg
```

Reply:
No worry, you did all the work there, just have to click one button ;)